internal/server/apimigrator.go

```diff
@@ -130,6 +130,12 @@ func rebuildRequest(c *gin.Context, newReqObj interface{}) {
 				query.Add(fieldName, fmt.Sprintf("%d", f.Int()))
 			case reflect.Uint, reflect.Uint64:
 				query.Add(fieldName, fmt.Sprintf("%d", f.Int()))
+			case reflect.Bool:
+				if f.Bool() {
+					query.Add(fieldName, "1")
+				} else {
+					query.Add(fieldName, "0")
```

Comment:
Hopefully it's not a problem that we use `1`/`0` here and  `true`/`false` in the `api.Client`. I guess `strconv.FormatBool` could be used here too. Not a blocker.